packages/patternfly-react/sass/patternfly-react/_patternfly-react.scss

```diff
@@ -5,6 +5,7 @@
 @import 'utilization-bar';
 @import 'breadcrumb';
 @import 'label-remove';
+@import 'masterhead';
 @import 'notificationdrawer';
 @import 'inline-edit';
 @import 'field-level-help';
```

Comment:
You will need to add the less version as well. Should convert easily, just changing the variables from $ to @